app/repeater/settings.py

```diff
@@ -35,7 +35,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(int(os.environ.get('DEBUG', 0)))
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
Shouldn't need this wildcard for development (as long as DEBUG is true, i think), and it shouldn't be included in production as far as I know.
Is there cause to include this, or should it be removed?

Reply:
it should be removed
but just to confirm, allow host is a list of domains that can call the api right?